packages/sanity/src/core/form/studio/tree-editing/components/search/TreeEditingSearchMenu.tsx

```diff
@@ -1,11 +1,59 @@
 import {Box, Button, Stack, Text} from '@sanity/ui'
 import {isEqual} from 'lodash'
 import {useCallback} from 'react'
-import {CommandList, type Path, supportsTouch, useTranslation} from 'sanity'
+import {
+  CommandList,
+  getSchemaTypeTitle,
+  type Path,
+  supportsTouch,
+  unstable_useValuePreview as useValuePreview,
+  useTranslation,
+} from 'sanity'
 
 import {type TreeEditingMenuItem} from '../../types'
 import {ITEM_HEIGHT} from './constants'
 
+interface ResultItemProps {
+  item: TreeEditingMenuItem
+  onPathSelect: (path: Path) => void
+  isFirst: boolean
+}
+
+function ResultItem(props: ResultItemProps): JSX.Element {
+  const {item, onPathSelect, isFirst} = props
+
+  const {value} = useValuePreview({
+    value: item.value,
+    schemaType: item.schemaType,
+  })
+
+  const title = value?.title || 'Untitled'
```

Comment:
Add the translation to "Untitled" here (and on the other instances where we have it)

[TreeEditingBreadcrumbsMenu.tsx](https://github.com/sanity-io/sanity/pull/6852/files#diff-b7e3ae4e075794063b24f74042a8fd13fef1afc913d42a28a994dda97ec34b6d), [TreeEditingMenu.tsx](https://github.com/sanity-io/sanity/pull/6852/files#diff-af0da0ddc1ba8d0d68ef788bfcd8ab8b38bc26d1d5e04cb0ceb192f19c8987b2), [TreeEditingBreadcrumbs.tsx](https://github.com/sanity-io/sanity/pull/6852/files#diff-8055b57f0590d5274115bb93340acef874cae5927c6c930bcd9bc08d3def4777)